brushfire-finatra/src/main/scala/com/stripe/brushfire/finatra/BrushfireServer.scala

```diff
@@ -1,12 +1,13 @@
 package com.stripe.brushfire.finatra
 
 import com.stripe.brushfire._
+import com.twitter.algebird.Semigroup
 import com.twitter.finatra._
 import com.twitter.finagle.http._
 import com.twitter.bijection._
 
 class BrushfireServer extends FinatraServer {
-  def score[K, V, T, P](root: String, trees: Iterable[Tree[K, V, T]], voter: Voter[T, P])(fn: ParamMap => Map[K, V]) = {
+  def score[K, V, T: Semigroup, P](root: String, trees: Iterable[Tree[K, V, T]], voter: Voter[T, P])(fn: ParamMap => Map[K, V]) = {
```

Comment:
why add the semigroup constraint? This seems like a semantic change rather than style/performance.